bucket/syncthingtray.json

```diff
@@ -0,0 +1,45 @@
+{
+    "version": "1.1.1",
+    "homepage": "https://github.com/Martchus/syncthingtray",
+    "description": "Tray application for Syncthing",
+    "license": {
+        "identifier": "GPL-2.0,...",
+        "url": "https://github.com/Martchus/syncthingtray/blob/master/LICENSES-windows-distribution.md"
+    },
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/Martchus/syncthingtray/releases/download/v1.1.1/syncthingtray-1.1.1-i686-w64-mingw32.exe.zip",
+            "hash": "802866f997c0bf03cc70f8969ce402036e6831ce94d7d63cb471e7be73fce98a"
+        },
+        "64bit": {
+            "url": "https://github.com/Martchus/syncthingtray/releases/download/v1.1.1/syncthingtray-1.1.1-x86_64-w64-mingw32.exe.zip",
+            "hash": "85c16af32fcafacc7e8d37609f0bdfb3b038e922728d51577083f47f9bbb3565"
+        }
+    },
+    "pre_install": "Move-Item \"$dir\\syncthingtray-*.exe\" \"$dir\\syncthingtray.exe\"",
+    "bin": [
+        [
+            "syncthingtray.exe",
+            "syncthingtray"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "syncthingtray.exe",
+            "Syncthing Tray"
+        ]
+    ],
+    "autoupdate": {
```

Comment:
```suggestion
    "checkver": "github",
    "autoupdate": {
```

Reply:
I guess this means `checkver` should come before `autoupdate` and simply "github" is enough? Done with the latest version of this PR.